.github/workflows/check-users-in-space-developer-role.yml

```diff
@@ -0,0 +1,70 @@
+on:
+  workflow_dispatch:
```

Comment:
thought you wanted to run this on schedule?

Reply:
correct... I wanted to do a dry run manually before scheduling.

Reply:
you can test on the same branch using the `on: push` trigger

Reply:
you can add **both** schedule and push triggers